collect_app/src/main/java/org/odk/collect/android/fragments/ShowQRCodeFragment.java

```diff
@@ -149,7 +149,12 @@ public void onClick(View v) {
             case R.id.btnSelect:
                 Intent photoPickerIntent = new Intent(Intent.ACTION_PICK);
                 photoPickerIntent.setType("image/*");
-                startActivityForResult(photoPickerIntent, SELECT_PHOTO);
+                if (photoPickerIntent.resolveActivity(getActivity().getPackageManager()) != null) {
+                    startActivityForResult(photoPickerIntent, SELECT_PHOTO);
+                } else {
+                    ToastUtils.showShortToast(getString(R.string.no_image_chooser));
+                    Timber.e(getString(R.string.no_image_chooser));
```

Comment:
It should be `Timber.w` it's not our fault that user doesn't have any external app to capture images.